File: Practica 1/Practica1.java
public class Practica1 {
    
    public static class Punto {
        public float x;
        public float y;
        
        public Punto(float x, float y){
            this.x = x;
            this.y = y;
        }
    
        public String coord_cartesianas(){
            return "(" + x +","+ y + ")";
        }
    
        public String coord_polares(){
            double r = Math.sqrt(x*x + y*y);
            double theta = Math.atan2(y,x);
            return ("Radio: "+ r +" Angulo: " + theta);
        }
        public String toString(){
            return ("Punto: " + x + ", " + y );
        }
        public static void main(String[] args){
            Punto p = new Punto(3, 4);
            System.out.println("Cartesianas: " + p.coord_cartesianas());
            System.out.println("Polares: " + p.coord_polares());
            System.out.println("Texto: " + p.toString());
        }
    }
}

File: Practica 1/Practica 1.py
import math
class Punto:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        return (self.x, self.y)
        
    def coord_polares(self):
        r = math.sqrt(self.x ** 2 + self.y ** 2)
        theta = math.atan2(self.y, self.x)
        return (f"Radio: {r}, Angulo: {theta}")
    
    def __str__(self):
        return f"Punto {self.x}, {self.y}"

p = Punto(3,4)
print("Cartesianas:", p.coord_cartesianas())  
print("Polares:", p.coord_polares())
print("Texto:", p.__str__())